marketplace/google-analytics-app/src/Timeline.tsx

```diff
@@ -0,0 +1,132 @@
+/* global gapi */
```

Comment:
We should instead pass `gapi` into the component through props. This way we can eliminate more global variables.